testsuite/run_sets/secondary_parallelizable.yml

```diff
@@ -40,7 +40,7 @@
 - features/secondary/srv_task_status_engine.feature
 - features/secondary/srv_errata_api.feature
 
-- features/secondary/proxy_retail_pxeboot_and_mass_import.feature
+#- features/secondary/proxy_retail_pxeboot_and_mass_import.feature
```

Comment:
Maybe add a comment, why this is disabled.

Reply:
Yup, I will add a TODO comment too here.

Reply:
Thank you.

Reply:
I just skipped the feature, depending the environment.